src/renderer/routes/test_sequencer_panel/components/dnd/DraggableRowSequence.tsx

```diff
@@ -0,0 +1,113 @@
+import { useDrag, useDrop } from "react-dnd";
+import {
+  Droppable,
+  ItemTypes,
+  TestSequenceDropResult,
+} from "@/renderer/routes/test_sequencer_panel/models/drag_and_drop";
+import { TableCell, TableRow } from "@/renderer/components/ui/table";
+import { Row, flexRender } from "@tanstack/react-table";
+import { parseInt } from "lodash";
+import { useTestSequencerState } from "@/renderer/hooks/useTestSequencerState";
+import { TestSequenceContainer } from "@/renderer/types/test-sequencer";
+
+export const DraggableRowSequence = ({
+  isSelected,
+  row,
+  ...props
+}: {
+  isSelected: boolean;
+  row: Row<TestSequenceContainer>;
+}): React.ReactNode => {
```

Comment:
no need to mark return type
props type shouldn't be inlined for style reasons (do `type Props = { ... }`)
useless spread operator (there's only 2 props and you already destructured them both, `...props` is completely empty)